README.md

```diff
@@ -13,28 +13,20 @@ pip install SimpleITK
 ## Installing SimpleITK for Python from the Python Packaging Source
 
 ```bash
-pip install SimpleITK
+pip install --no-binary :all: SimpleITK
 ```
 
 ### Prerequisites
 
-Building *requires*:
-* [CMake](https://cmake.org)
+The build requirements are specified in the pyproject.toml file via [PEP 518](https://peps.python.org/pep-0518/). The should be automatically downloaded and when using a [PEP 517](https://peps.python.org/pep-0517/) compliant build front-end.
```

Comment:
"The should be automatically downloaded and when using..." -> "They should be automatically downloaded when using..."